qc_ositrace/checks/deserialization/deserialization_checker.py

```diff
@@ -68,7 +68,10 @@ def run_checks(config: Configuration, result: Result) -> None:
     try:
         try:
             trace = OSITrace(
-                config.get_config_param("InputFile"), config.get_config_param("osiType")
+                config.get_config_param("InputFile"),
+                config.get_config_param("osiType"),
+                False,
+                config.get_config_param("osiTopic"),
```

Comment:
```suggestion
                path= config.get_config_param("InputFile"),
                type_name = config.get_config_param("osiType"),
                cache_messages = False,
                topic = config.get_config_param("osiTopic"),
```

I know you wanted a review that it works but I could not resist to also look at the code ;) 
As we just changed the interface and this priject also follows the latest osi python main I would recommend to change to keywords (to even fail a little bit earlier and more clear).